tests/test_exports.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+from django_prometheus.exports import SetupPrometheusEndpointOnPortRange
+import unittest
+
+
+class ExportTest(unittest.TestCase):
+    def testPortRange(self):
+        port_range = [8000, 8001]
```

Comment:
I'm not a fan of the fact that this relies on fixed ports that may not be available under test. Can you mock HTTPServer in this test, to ensure this behaves as intended? Sorry, I know this makes the test a lot more complicated, but I really like my tests to be isolated from the underlying system whenever possible :)